### ISSUE TITLE
RubyNLP

### ISSUE THREAD

**Issue by arbox** [solyaris/naif#1]
Hi Giorgio,

adding the `rubynlp` GitHub topic helps meanwhile to discover NLP related repositories :)

Best,
Andrei

**Comment by solyarisoftware** [comment]
Hi Andrei!
you are right! I just added the tag you suggest.

BTW, unfortunately I didn't (yet) published the Ruby implementation code; for a bunch of reasons maybe i can share via email (giorgio.robino@mail.com)

Anyway I'll glad to read your comments on slides:
https://convcomp.it/naif-ruby-micro-framework-to-build-dumb-chat-machines-5c552a8c8f7e

Last but not least, I actively support [ChatScript](https://github.com/bwilcox-1234/ChatScript/) project and I released also a small Ruby client interface as gem: https://github.com/solyaris/rChatScript/

I just added the tag also there

thanks & regards
giorgio
